x-pack/test/functional/apps/saved_objects_management/feature_controls/saved_objects_management_security.ts

```diff
@@ -245,9 +245,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       // From https://github.com/elastic/kibana/issues/59588 edit view became read-only json view
       // test description changed from "edit" to "inspect"
       // Skipping the test to allow code owners to delete or modify the test.
```

Comment:
AFAIK, the comment refers to the "edit view", which is still skipped because it's no longer a feature (line 279 onwards). Happy to remove the comment and the test, though :)